planning_launch/config/adaptive_cruise_control.yaml

```diff
@@ -0,0 +1,36 @@
+/**:
+  ros__parameters:
+    # Adaptive Cruise Control (ACC) config
```

Comment:
Refer this.
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/b0ba4f052ed836f5bc69e87d7f3ca83fec37f8e2/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/config/adaptive_cruise_control.yaml#L3